data/razer

```diff
@@ -7,3 +7,8 @@ razerzone.com @cn
 razerzone.jp
 respawnbyrazer.com
 rzr.to
+
+full:b6y6lf1s29.execute-api.us-east-1.amazonaws.com
+
+# tracking
+full:o72136.ingest.sentry.io @ads
```

Comment:
Maybe you can move this domain to `razer-ads` and include it in this category, and `category-ads`.